<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>738</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame_4">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_central">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QFrame" name="frame_left">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>30</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QSplitter" name="splitter_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QFrame" name="frame_fileSystem">
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="midLineWidth">
                <number>0</number>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <property name="bottomMargin">
                 <number>20</number>
                </property>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Group</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2" colspan="3">
                 <widget class="QLabel" name="label_7">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Variable</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="comboBox_group">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="currentIndex">
                   <number>-1</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" colspan="3">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="2" column="5">
                 <widget class="QComboBox" name="comboBox_variable">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>File System</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="6">
                 <widget class="QListWidget" name="listWidget_file">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" colspan="6">
                 <layout class="QVBoxLayout" name="lcomboBox_globalProcessing"/>
                </item>
               </layout>
              </widget>
              <widget class="QTabWidget" name="tabWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="tab_3">
                <attribute name="title">
                 <string>Parameters</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QFrame" name="frame_parameters">
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Sunken</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_2">
                    <property name="leftMargin">
                     <number>2</number>
                    </property>
                    <property name="topMargin">
                     <number>2</number>
                    </property>
                    <property name="rightMargin">
                     <number>2</number>
                    </property>
                    <property name="bottomMargin">
                     <number>2</number>
                    </property>
                    <item>
                     <layout class="QVBoxLayout" name="layout_parameters"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_4">
                <attribute name="title">
                 <string>Informations</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QTreeWidget" name="treeWidget">
                   <column>
                    <property name="text">
                     <string>Labels</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Data</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="text">
                    <string>Save</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>Individual Processing</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QVBoxLayout" name="layout_singleProcessing"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="GlobalProcessing">
                <attribute name="title">
                 <string>Global Processing</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <layout class="QVBoxLayout" name="layout_globaProcessing"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab">
                <attribute name="title">
                 <string>.</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_4"/>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QFrame" name="frame_right">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QFrame" name="frame_timeline">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>45</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>46</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="leftMargin">
                <number>11</number>
               </property>
               <property name="topMargin">
                <number>11</number>
               </property>
               <property name="rightMargin">
                <number>11</number>
               </property>
               <property name="bottomMargin">
                <number>11</number>
               </property>
               <item>
                <widget class="QLabel" name="label_timeline_dim">
                 <property name="text">
                  <string>Channel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_dim"/>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_layout_display">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>70</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <property name="leftMargin">
                <number>5</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <property name="spacing">
                <number>3</number>
               </property>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="layout_log"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionRemove"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuLayout">
     <property name="title">
      <string>Layout</string>
     </property>
     <addaction name="action1_windows"/>
     <addaction name="action2_windows"/>
     <addaction name="action3_windows"/>
     <addaction name="action4_windows"/>
    </widget>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New +</string>
     </property>
     <addaction name="actiontype_1"/>
     <addaction name="actiontype_2"/>
     <addaction name="actiontype_3"/>
     <addaction name="actiontype_4"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="menuLayout"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionRun_Analysis"/>
    <addaction name="actionOpen_Analysis"/>
   </widget>
   <addaction name="menufiles"/>
   <addaction name="menuWindow"/>
   <addaction name="menuAnalysis"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>Create New</string>
   </property>
  </action>
  <action name="actionRun_Analysis">
   <property name="text">
    <string>Run Analysis</string>
   </property>
  </action>
  <action name="action1_windows">
   <property name="text">
    <string>1 window</string>
   </property>
  </action>
  <action name="action2_windows">
   <property name="text">
    <string>2 windows</string>
   </property>
  </action>
  <action name="action3_windows">
   <property name="text">
    <string>3 windows</string>
   </property>
  </action>
  <action name="action4_windows">
   <property name="text">
    <string>4 windows</string>
   </property>
  </action>
  <action name="actiontype_1">
   <property name="text">
    <string>type 1</string>
   </property>
  </action>
  <action name="actiontype_2">
   <property name="text">
    <string>type 2</string>
   </property>
  </action>
  <action name="actiontype_3">
   <property name="text">
    <string>type 3</string>
   </property>
  </action>
  <action name="actiontype_4">
   <property name="text">
    <string>type 4</string>
   </property>
  </action>
  <action name="actionManage">
   <property name="text">
    <string>Manage</string>
   </property>
  </action>
  <action name="actionOpen_Analysis">
   <property name="text">
    <string>Open Analysis</string>
   </property>
  </action>
  <action name="actionApply">
   <property name="text">
    <string>Apply</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove all</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
